homeassistant/components/isy994/binary_sensor.py

```diff
@@ -425,6 +428,12 @@ async def async_added_to_hass(self) -> None:
         # Start the timer on bootup, so we can change from UNKNOWN to OFF
         self._restart_timer()
 
+        if (last_state := await self.async_get_last_state()) is not None:
+            # Only restore the state if it was previously ON (Low Battery)
+            if last_state.state == STATE_ON:
+                self._computed_state = True
```

Comment:
```suggestion
        if (last_state := await self.async_get_last_state()) is not None and last_state.state == STATE_ON:
            # Only restore the state if it was previously ON (Low Battery)
            self._computed_state = True
```

Could save some indent here but doesn't make much difference as its not complex

Reply:
I had that but changed it--for some reason I thought Pylint got mad about `:=` and a second reference in the same line.

Reply:
Black splits it to 3 lines, so 6/half-dozen.